build-repo.sh

```diff
@@ -46,6 +47,36 @@ case "${VERSION_PER_SUBPACKAGE}" in
     ;;
 esac
 
+case "${ENABLE_CCACHE}" in
+  "")
+    ENABLE_CCACHE=no
+    ;;
+  yes)
+    ;;
+  no)
+    ;;
+  *)
+    echo "ENABLE_CCACHE must be one of: \"yes\", \"no\", \"\" (default: \"no\")"
+    exit 1
+    ;;
+esac
+
+case "${SKIP_ROSDEP_UPDATE}" in
+  "")
+    SKIP_ROSDEP_UPDATE=false
+    ;;
+  yes|true)
+    SKIP_ROSDEP_UPDATE=true
+    ;;
+  no|false)
+    SKIP_ROSDEP_UPDATE=false
+    ;;
+  *)
+    echo "ENABLE_CCACHE must be one of: \"yes\", \"no\", \"\" (default: \"no\")"
```

Comment:
I think it should be `SKIP_ROSDEP_UPDATE` here and not `ENABLE_CCACHE`. We could also mention `true` and `false` as valid values.

Reply:
I think it's better to unify all options to yes/no instead of true/false. (some of the other options are already using yes/no)
true/false for `SKIP_ROSDEP_UPDATE` is for backward compatibility.